os/customizing-docker.md

```diff
@@ -66,6 +66,123 @@ To keep access to the port local, replace the `ListenStream` configuration above
         ListenStream=127.0.0.1:2375
 ```
 
+## Enable the Remote API with TLS authentication
+
+Docker doesn't support HTTPS on systemd's `fd://` socket so it is impossible to configure socket activated docker service with TLS auth.
+
+Docker TLS configuration consists of two parts: keys creation and systemd drop-in configuration.
+
+### TLS keys creation
+
+There are lots of instructions on how to create TLS keys. In this example we will show you how to create TLS keys with [etcd-ca](https://github.com/coreos/etcd-ca) tool. Please note that if you want to increase security you have to enter passphrase for the key. We assume that you will create keys on CoreOS host and then copy client's certificates to local machine.
+
+```sh
+# Initialize Certificate Authority
+./etcd-ca init --passphrase=""
+# Create certificate request for %hostname% docker server
+./etcd-ca new-cert --passphrase="" --domain="example.com" --ip="%private_ip%,%public_ip%" %hostname%
```

Comment:
why the mixes of example.com and %hostname%, is the idea that %hostname% would be somehost.example.com? Also I generally prefer example values to substitution values, otherwise a user may think that the tool will perform the substitution. So `--ip="192.0.2.10,98.51.100.11"`